about.js

```diff
@@ -2,7 +2,7 @@
  * About this module FormApp
  * @extends FormApplication
  */
-class PinCushionAboutApp extends FormApplication {
+export class PinCushionAboutApp extends FormApplication {
```

Comment:
Since this is the only export in this file, can you set it as `default` and adjust the import accordingly?